lib/runtime/goto.js

```diff
@@ -2,20 +2,55 @@
 
 import path from 'path'
 import fs from 'fs'
+import { CompositeDisposable } from 'atom'
 
 import { client } from '../connection'
 import modules from './modules'
 import { isValidScopeToInspect } from '../misc/scopes'
 import { getWord, getWordRangeAtBufferPosition, isValidWordToInspect } from '../misc/words'
+import { getLocalContext } from '../misc/blocks'
 
-const methods = client.import('methods')
+const goto = client.import('goto')
 
+const grammar = atom.grammars.grammarForScopeName('source.julia')
```

Comment:
This *needs* to be in `activate` or wherever it is actually used, otherwise it'll break if language-julia is loaded after julia-client (and that ordering isn't stable).

Reply:
Ah, I've forgotten that. Thanks for reminding me :)
Will fix soon.